js/deps.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as msg from "gen/cli/msg_generated";
+import { sendSync } from "./dispatch";
+import { decodeMessage } from "./workers";
+import { assert } from "./util";
+import * as flatbuffers from "./flatbuffers";
+
+export function deps(): any {
```

Comment:
def want to come back and drop this `any` return type/create Interface for deps, etc